openid-4-verifiable-presentations-1_0.md

```diff
@@ -336,7 +341,7 @@ The following is a non-normative example of a transaction data content, after ba
 The following additional considerations are given for pre-existing Authorization Request parameters:
 
 `nonce`:
-: REQUIRED. Defined in [@!OpenID.Core]. It is used to securely bind the Verifiable Presentation(s) provided by the Wallet to the particular transaction. See (#preventing-replay) for details. Values MUST only contain ASCII URL safe characters (uppercase and lowercase letters, decimal digits, hyphen, period, underscore, and tilde).
+: REQUIRED. A case-sensitive String generated from at least 16 fresh, cryptographically random bytes. It is used to securely bind the Verifiable Presentation(s) provided by the Wallet to the particular transaction. See (#preventing-replay) for details. Values MUST only contain ASCII URL safe characters (uppercase and lowercase letters, decimal digits, hyphen, period, underscore, and tilde). If OpenID Connect is used, refer to its definition in [@!OpenID.Core].
```

Comment:
This is technically a normative change, the existing text about nonce being at least 16 fresh cryptographically random bytes is currently only in implementation considerations.

Reply:
It is related since there was a concern that for DC API, the definition of nonce in OIDC does not make so much sense.